packages/toggleBox/stories/helpers/ToggleBoxGroupStoryHelper.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from "react";
+
+interface ToggleBoxGroupStoryHelperState {
+  selectedItems: string[];
+}
+
+interface RenderProps {
+  changeHandler: (selectedItems: string[]) => void;
+  selectedItems: string[];
+}
+interface ToggleBoxGroupStoryHelperProps {
+  children: (renderProps: RenderProps) => React.ReactNode;
+  selectedItems?: string[];
+}
+
+class ToggleBoxGroupStoryHelper extends React.PureComponent<
```

Comment:
No, we're not committed to class components. We have a few stateless functional components.

I just haven't used hooks yet because the React versions we specify in `peerDependencies` include versions of React that don't have hooks yet.